roles/Cargo.toml

```diff
@@ -22,7 +22,7 @@ members = [
     "test-utils/sv1-mining-device",
     "translator",
     "jd-client",
-    "jd-server",
+    "jd-server", "tests-integration",
```

Comment:
It's a stupid thing, but could you put it under `jd-server` like all the others?

Reply:
done